server/routerlicious/packages/services-shared/src/runner.ts

```diff
@@ -103,7 +110,7 @@ export function runService<T extends IResources>(
 
 	const waitInMs = waitBeforeExitInMs ?? 1000;
 	const runnerMetric = Lumberjack.newLumberMetric(LumberEventName.RunService);
-	const runningP = run(config, resourceFactory, runnerFactory, logger);
+	const runningP = run(config, resourceFactory, runnerFactory, logger, secretsList);
```

Comment:
Any reason why we don't call config dumper here, instead of passing it to the `run()` method? if there is no specific reason, maybe we could do it here instead?